types/vergeSense/cloudConnector/space_report.schema.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$id": "https://akenza.io/vergeSense/cloudConnector/space_report.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "space_report",
+  "title": "Space report",
+  "properties": {
+    "personCount": {
```

Comment:
align this with area count? or is it more people flow?

Reply:
Its a readout of the motions captured in the timeframe.